docker/release/nanoserver/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# escape=`
```

Comment:
@andschwa  I've created in-line script versions of the Linux nightly builds here:  https://github.com/brycem/PowerShell/tree/docker/docker/nightly  Example:  'docker build https://raw.githubusercontent.com/brycem/PowerShell/docker/docker/nightly/centos7/Dockerfile'